mantidproject/mantid
# Set the source dependencies of generated sip code correctly

Description of work.

Fixes the dependency chain between the sip code and the final generated module. Previously any changes in the sip source was not causing the sip code generator to run again. This fixes that.

**To test:**

* Before merging the branch open `qt/python/mantidqtpython/mantidqtpython_def.sip` and make a minor change that should cause a syntax error and save the file. Run the build and see that the sip code generator is not run.
* Merge the branch and do the same and the full chain from sip code generation to build should now complete.  

*No issue*

**Release Notes** 
*Does not need to be in the release notes.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
